### PR TITLE
Learn factory logo added

### PR DESCRIPTION
**Pull request by siccoo**

### CODE REVIEW

**Comment by Odusanya** [comment]
Hey Michael, @siccoo  
Thanks for contributing to this. Please make sure your assets are well optimized, according to the recommendations made [here](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md#sending-in-a-pull-request)

**Comment by siccoo** [comment]
> Hey Michael, @siccoo
> Thanks for contributing to this. Please make sure your assets are well optimized, according to the recommendations made [here](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md#sending-in-a-pull-request)

Yeah Odusanya they are optimized as directed; as well as the logos.json file all optimized

**Comment by Odusanya** [comment]
> > Hey Michael, @siccoo
> > Thanks for contributing to this. Please make sure your assets are well optimized, according to the recommendations made [here](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md#sending-in-a-pull-request)
> 
> Yeah Odusanya they are optimized as directed; as well as the logos.json file all optimized

Got it. Please select the minify option on SVGOMG too, so the tool can remove all unnecessary white space and comments. See screenshot ->
image

**Comment by siccoo** [comment]
> > > Hey Michael, @siccoo
> > > Thanks for contributing to this. Please make sure your assets are well optimized, according to the recommendations made [here](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md#sending-in-a-pull-request)
> > 
> > 
> > Yeah Odusanya they are optimized as directed; as well as the logos.json file all optimized
> 
> Got it. Please select the minify option on SVGOMG too, so the tool can remove all unnecessary white space and comments. See screenshot ->
> image

Okay @Odusanya I've minified all logos through that tool. Thank you

File: logos/amplify/amplify.svg [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+HLYXGYpYMfvY7T5SM+YThumvSHGT3UTR1IlA3MzP5qAXbZs3DVc6uczQ3A1sM+xaDGUXTkW8Qwuz
+E8Vc2tF1TGzLZZPzhvFMki0Sc7YbzF6MyXGUqfSzbt9YXLqzkWWPWpQyNooWaXUozAazrDk4urOS
++CaGXqAIbfvLPDD7hIc6g+WXVBZYN+FtpYi0rLpqRhpBxU33vMGChvBBQ/igIXzQED5oCB80hA8a
+wgcN4YOG8EFD+KAhfNAQPmgIHzSEDxrCBw3hg4bwQUP4oCF80BA+aAgfNIQPGsIHDeGDhvBBQ/ig
+IXzQED5oCB80hA8awgcN4YOG8EFD+KAhfNAQPmgIHzSEDxrCBw3hg4bwQUP4oCF8/heGvPm3Wo/F
+D++fPx+bn/8CeIODmnCqdDIAAAAASUVORK5CYII=
+"
+     id="image10"
+     x="0"
+     y="0" />
```
Comment (Odusanya) [inline logos/amplify/amplify.svg]: Also, your SVGs should contain pure vectors. 
You have uploaded pngs and wrapped as SVGs, as you can see from the `<image` tag highlighted. This isn't pure vector and as a result, you can't edit Colours and paths in vector tools like illustrator, etc.

Please replace your logos with vector

Comment (siccoo) [reply logos/amplify/amplify.svg]: Okay I'm using inkscape here so I saved it as SVG, what other vector tool would you suggest I use?

**Comment by demilad** [comment]
Hey @siccoo thanks for your contribution.
I noticed you've updated the svg logos to what appears to be vector image traces with poor quality.
To ensure high quality logos, I'll suggest you either reach out to the logo designers for the original vector files or recreate the logos from scratch in a vector tool l like Inkscape or Illustrator (requires more effort as you have to make a pixel perfect recreation for your contribution to be added)

**Comment by siccoo** [comment]
> Hey @siccoo thanks for your contribution.
> I noticed you've updated the svg logos to what appears to be vector image traces with poor quality.
> To ensure high quality logos, I'll suggest you either reach out to the logo designers for the original vector files or recreate the logos from scratch in a vector tool l like Inkscape or Illustrator (requires more effort as you have to make a pixel perfect recreation for your contribution to be added)

Okay I will @demilad. Thank you

**Comment by Odusanya** [comment]
Hey @siccoo did you run this locally, just tested and it appears your image path is broken(see screenshot below). 
Please make a fresh PR using all the learnings from this PR, test locally to ensure everything works and then make a new PR.

I'll be closing this one now. Thanks for your contribution, looking forward to your new PR.

image

**Pull request closed without merging**